recipes/procps-ng/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = '3.3.15' %}
+
+package:
+  name: 'procps-ng'
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/procps-ng/procps/-/archive/v{{ version }}/procps-v{{ version }}.tar.gz
+  sha256: 14dfa751517dd844efa9f492e3ad8071f908a269c6aea643b9a1759235fa2053
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - autoconf
+    - automake
+    - libtool
+    - make
+    - pkg-config
+  host:
+    - ncurses
+  run:
+    - ncurses
+
+test:
+  commands:
+    - ps
```

Comment:
[This comment](https://github.com/conda-forge/staged-recipes/pull/3958#discussion_r145011560) might be a good idea to avoid testing with the host's binaries.

Reply:
Good point, yeah we should add `$PREFIX`